requirements-dev.txt

```diff
@@ -5,9 +5,11 @@ ruff-lsp==0.0.39
 mypy==1.5.1
 pytest==7.4.2
 pytest-asyncio==0.23.2
+pytest-xdist==3.5.0
```

Comment:
This takes care of multiprocessing, not async, right? (Seems like that's [what the package says](https://github.com/pytest-dev/pytest-xdist)!) Isn't Async done through using the _ask_llm_async function?

Reply:
It allows you to run multiple processes. `pytest-asyncio` allows you to call `async` functions. And the core call to the llm `_ask_llm_async` is indeed `async`